include/drivers/led.h

```diff
@@ -73,7 +73,7 @@ __subsystem struct led_driver_api {
  * This routine starts blinking an LED forever with the given time period
  *
  * @param dev LED device
- * @param led LED channel/pin
+ * @param led LED channel/pin/id
```

Comment:
This part I think conflicts with #26101 here:

https://github.com/zephyrproject-rtos/zephyr/pull/26101/commits/db8213c76ed2414be464a902c0af4495206840f3

(The shortlog is ` led: move API functions to LED objects` in case that commit SHA goes stale due to rebasing.)

I honestly like the #26101 phrasing a bit better. Could we adopt that here?